documentation/docs/steps/apiProviderDownload.md

```diff
@@ -0,0 +1,30 @@
+# ${docGenStepName}
+
+## ${docGenDescription}
+
+## Prerequisites
+
+## ${docGenParameters}
+
+## ${docGenConfiguration}
+
+## ${docJenkinsPluginDependencies}
+
+## Example
```

Comment:
Documentation is the face to the customer. The documentation should not only provide the arguments of a step and its heading, as guaranteed by the configuration yaml. It should also describe scenarios in which the step is used. And any problems. Each gap means trial and error for the customer. Curious Experts look into the code to understand, laymen generate an open issue in the repository, in the worst case even in Service Now. So many others could get involved.

Reconsider:
 any Prerequisites?
 expected scenarios?
 problems the consumer could face?
 useful links?